release-content/release-notes/atmosphere_environment_map.md

```diff
@@ -1,16 +1,15 @@
 ---
-title: Generated environment map for procedural atmosphere
+title: Generate environment maps from the procedural atmosphere
 authors: ["@mate-h"]
 pull_requests: [20529]
 ---
 
 (TODO: Embed screenshot of atmosphere-generated reflections)
 
-You can now have dynamic reflections and ambient light in your scene that match the procedural sky.
+As your procedural sky changes, the reflections and ambient light in your scene should automatically be updated to match.
+Now, they can be — no pre-baked environment maps needed.
```

Comment:
The two sentence split here is a little weird. Maybe talk about how there was no option in Bevy 0.16?

Reply:
```suggestion
Now, this just works!
This is fully dynamic: no pre-baked environment maps are needed.
```